lib/ansible/modules/cloud/amazon/iam.py

```diff
@@ -554,12 +555,12 @@ def create_role(module, iam, name, path, role_list, prof_list, trust_policy_doc)
         if name not in role_list:
             changed = True
             iam_role_result = iam.create_role(name,
-                assume_role_policy_document=trust_policy_doc,
-                path=path).create_role_response.create_role_result.role
+                                              assume_role_policy_document=trust_policy_doc,
+                                              path=path).create_role_response.create_role_result.role
 
             if name not in prof_list:
                 instance_profile_result = iam.create_instance_profile(name,
-                    path=path).create_instance_profile_response.create_instance_profile_result.instance_profile
+                                                                     path=path).create_instance_profile_response.create_instance_profile_result.instance_profile
```

Comment:
Long method names like this put the style checkers in a bind.  Here's what I'd recommend:

``` python
                instance_profile_result = iam.create_instance_profile(name, path=path) \
                        .create_instance_profile_response.create_instance_profile_result.instance_profile
```

I hate to break on the dot but in cases like these, I think it's the best of several bad options.

Reply:
I tend to do something like:

```
                instance_profile_result = iam.create_instance_profile(
                    name,
                    path=path
                ).create_instance_profile_response.create_instance_profile_result.instance_profile
```

Just an FYI, figured I'd chime in.